packages/client/tests/functional/extended-where/findUnique.ts

```diff
@@ -0,0 +1,38 @@
+import testMatrix from './_matrix'
+import { setup } from './_setup'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+let vars: Awaited<ReturnType<typeof setup>>
+testMatrix.setupTestSuite(() => {
+  beforeAll(async () => {
+    vars = await setup(prisma)
+  })
+
+  test('findUnique with where 1 unique (PK)', async () => {
+    await prisma.user.findUnique({
```

Comment:
Shouldn't we assert something here?

Reply:
Yes, I think I should assert the results